From 6a2c90127ab305604fa51f30bce4f23179fa4b63 Mon Sep 17 00:00:00 2001
From: JamesH65 <JamesH65@users.noreply.github.com>
Date: Mon, 6 Feb 2017 15:24:47 +0000
Subject: [PATCH] gpio_mem: Remove unnecessary dev_info output (#1830)

The open function was spamming syslog every time
called, so have removed call completely.
---
 drivers/char/broadcom/bcm2835-gpiomem.c | 2 --
 1 file changed, 2 deletions(-)

--- a/drivers/char/broadcom/bcm2835-gpiomem.c
+++ b/drivers/char/broadcom/bcm2835-gpiomem.c
@@ -76,8 +76,6 @@ static int bcm2835_gpiomem_open(struct i
 	int dev = iminor(inode);
 	int ret = 0;
 
-	dev_info(inst->dev, "gpiomem device opened.");
-
 	if (dev != DEVICE_MINOR) {
 		dev_err(inst->dev, "Unknown minor device: %d", dev);
 		ret = -ENXIO;
